### DISCUSSION TITLE
【自定义顶栏的改进建议】

### DISCUSSION THREAD

**Discussion by silv3rarr0w** [the1812/Bilibili-Evolved#4949] [功能建议 / Ideas]
1. 【字体辨识度问题】自定义顶栏上去除了图标，但使用过程中图标的辨识度要比纯文字高很多，这导致在原版中瞟一眼就能点到想要的，而在自定义顶栏中还得额外确认一下，很不流畅，建议也把图标加上。
2. 【历史记录同步问题】自定义顶栏的历史记录无法自动同步，有时候还得手动刷新一下才能看到最新的，希望将来可以改进。
3. 【误触问题】右上角几个功能键，鼠标悬停到触发的时间间隔太短，导致很多时候我只是鼠标经过，就误触了。原版顶栏的触发间隔更长，使用过程中很少误触。希望可以把触发间隔加长一下。
4. 【历史记录删除问题】搜索栏的搜索记录右边有一个叉，点了之后搜索栏也会跟着消失，这导致想要删除几个特定条目的时候很麻烦，删除一个后还要额外再点一下搜索栏，才能删除下一个。一般的搜索栏在删除特定条目后都不会消失，希望后面可以改进。
5. 【其它】原版搜索栏在输入内容后，最右边会出现一个叉，方便删除，这在一些情况下很有用，希望可以把这个功能也加上。

**Comment by Geeyun-JY3** [comment]
1、（既然已经为此开过一个讨论 #4946，那下次还是不要再写进新 issue 了，重复的讨论只会增加负担 ；另外我个人是建议下次不要再开这种一次提多个意见的讨论，一个讨论专注一个主题比较好）`自定义顶栏`不是去除了图标，而是因为`自定义顶栏`仿的是旧版原版顶栏，而且图标这事之前也聊过：https://github.com/the1812/Bilibili-Evolved/discussions/2798#discussioncomment-1887160
4、看来 #4732 没修完
5、重复，请善用搜索：#801 的第 3 点、#4793

**Reply by Geeyun-JY3** [reply]
4. 通过 <kbd>/</kbd> 唤起的搜索栏也有一样的毛病
